src/styles/pages/main/main.scss

```diff
@@ -0,0 +1,377 @@
+.header {
+
+    &__logo {
+        padding: 9px 0;
+    }
+
+    .logo {
+        font-size: 35px;
+        color: white;
+        font-family: "Dancing Script", cursive;
+        border-bottom: 1px solid #fff;
+    } 
+
+    .container {
+        max-width: 1170px;
+    }
+
+    &__wrapper {
+        width: 100%;
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+        background-color: black;
+        position: fixed;
+    }
+
+    &__menu {
+        display: flex;
+        justify-content: space-between;
+        margin: 20px 0;
+
+        li {
+            padding: 0 30px;
+
+            a {
+                font-size: 20px;
+                color: white;
+                transition: all 0.5s ease;
+                border-bottom: 1px solid transparent;
+
+                &:hover {
+                    border-bottom: 1px solid #fff;
+                }
+            }
+        }
+    }
+
+    &__btn {
+        background-color: black;
+        border: 1px solid white;
+        border-radius: 10px;
+
+        &:hover {
+            background-color: grey;
+        }
+
+        button {
+            color: white;
+            font-size: 20px;
+            width: 90px;
+            padding: 10px;
+        }
+    }
+}
+
+.banner {
+    height: 100vh;
+
+    .background-image {
+        position: absolute;
+        inset: 0;
+        width: 100%;
+        height: 100%;
+        background-position: center;
+        background-size: cover;
+        background-repeat: no-repeat;
+        z-index: -1;
+        top: 58px;
+    
+        &::before {
+            content: "";
+            width: 100%;
+            height: 100%;
+            position: absolute;
+        }
+    
+        &_main   {
+            background-image: url(https://kartinki.pics/uploads/posts/2021-04/1617313820_4-p-foni-dlya-menyu-5.jpg) !important;
+        }
+    }
+    
+    .major {
+
+        .container {
```

Comment:
delete it when you move the container into common file
and add wrapper for paddings